src/evaluate.py

```diff
@@ -118,11 +119,11 @@ def evaluate(
         # task_preds will be a DataFrame with columns
         # ['preds'] + FIELDS_TO_EXPORT
         # for GLUE tasks, preds entries should be single scalars.
-
         # Update metrics
         task_metrics = task.get_metrics(reset=True)
         for name, value in task_metrics.items():
             all_metrics["%s_%s" % (task.name, name)] = value
+
         all_metrics["micro_avg"] += all_metrics[task.val_metric] * n_examples
```

Comment:
can you use the `task.contributes_micro_macro_avg` here as a check ? Also, consider renaming to something like `task.contributes_to_aggregate_score`?

Reply:
Yeah, that should make it clearer.